### DISCUSSION TITLE
Dots at start or end of line - Engraving - CO2 Laser

### DISCUSSION THREAD

**Discussion by mati181818** [arkypita/LaserGRBL#1216] [Q&A]
Hello.
Sorry if it is not ok in "issues", but I don't know how can I solve the problem.

My machine: 

 Load image in lasergrbl
 Set M4 mode to turn on the laser 
 Start engraving

I am using M4 mode, with laser mode enabled, and tried a lot of min pwm settings, but the results are almost the same.
![20201220_133355](https://user-images.githubusercontent.com/48954667/102719237-b1941b80-42cb-11eb-9a82-2492da9a233b.jpg)

Hardware and software configuration:
 - Windows 10
 - A homemade co2 50W laser.
 - Grbl version (1.1f)
 - LaserGRBL version: 3.8.5

How can I solve this?
Thanks!!

**Comment by arkypita** [comment]
Not a lasergrbl issue but an hardware issue, moved to [discussion](https://github.com/arkypita/LaserGRBL/discussions)

**Reply by Elieelienas** [reply]
hello @arkypita ,  please can you let me know the code (custom button) if i want to restart my burn image but from specific point not from the beginning of the image.  i am using the software laserGRBL v3.80  machine Ortur (laser20w)
